modules/cnv-edit-boot-order-web.adoc

```diff
@@ -0,0 +1,17 @@
+// Module included in the following assemblies:
+//
+// * cnv/cnv_users_guide/cnv-edit-boot-order.adoc
+
+[id="cnv-edit-boot-order-web_{context}"]
+= Editing a boot order list in the web console
+
+Edit the boot order list in the web console.
+
+.Procedure
+
+. Click *Workloads* -> *Virtual Machines* from the side menu.
+. Select a virtual machine to open the *Virtual Machine Overview* screen.
+. Click the pencil icon for *Boot Order*.
+. In the boot order list, hover over the arrow icon next to the item that you want to move, drag the item up or down, and drop it in a location of your choice.
+. If you use a screen reader, tap the *Up* or *Down* key to move the item in the boot order list. Then, tap the *Tab* key to drop the item in a location of your choice.
```

Comment:
If these are two different ways to do the same step, they should be presented more like this:

. Choose the appropriate method to move the item in the boot order list:
* If you use a screen reader, tap...
* If you do not use a screen reader, hover...

Reply:
This is still presented as two different steps.